tests/e2e/e2e_test.go

```diff
@@ -206,7 +210,8 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 			status, err := cli.Status(cctx)
 			ccancel()
 			gomega.Ω(err).Should(gomega.BeNil())
-			existingSubnetID = status.ClusterInfo.GetSubnets()[0]
+			customChains := status.ClusterInfo.GetCustomChains()
+			existingSubnetID = customChains[createdBlockchainID].SubnetId
```

Comment:
should you check that createdBlockchainID is a key in customChains? what happens if not? I think you will get a runtime error